### ISSUE TITLE
Interpolate on simple value does not take the value from the target

### ISSUE THREAD

**Issue by gaou-piou** [tweenrex/tweenrex#11]
Look at this example
https://codepen.io/gaougalos/pen/BmENMP

**Comment by notoriousb1t** [comment]
I see the issue.  https://github.com/tweenrex/render/blob/master/src/adapters/styleAdapter.ts#L5  Should really be 

```
    get(target: HTMLElement, prop: string) {
        return getComputedStyle(target)[prop]
    },
```

So unless opacity is explicitly being set on style, it isn't picking it up.  


I'll put in a fix for this later today.

**Comment by notoriousb1t** [comment]
This should be fixed in 1.4.2.  Let me know if that solves the issue 😄

**Comment by gaou-piou** [comment]
perfect
